src/nvim/vim.h

```diff
@@ -332,20 +332,15 @@ enum {
 /* Maximum number of bytes in a multi-byte character.  It can be one 32-bit
  * character of up to 6 bytes, or one 16-bit character of up to three bytes
  * plus six following composing characters of three bytes each. */
-# define MB_MAXBYTES    21
+#define MB_MAXBYTES    21
 
 /* This has to go after the include of proto.h, as proto/gui.pro declares
  * functions of these names. The declarations would break if the defines had
  * been seen at that stage.  But it must be before globals.h, where error_ga
  * is declared. */
```

Comment:
@Pyrohh Is it a comment for you or should I transform this comment to 

```
/* It must be before globals.h, where error_ga is declared. */
```

(even if not related with this PR)